templates/express_mysql/config/appConfig.js

```diff
@@ -1,18 +1,21 @@
+import { configDotenv } from 'dotenv'
+configDotenv({ path: "./.env" })
```

Comment:
prefer to import and use this in index.js. It'll be available here as this module will be part of the server package.